struct.go

```diff
@@ -25,7 +25,7 @@ type EurekaConnection struct {
 	DNSDiscovery   bool
 	DiscoveryZone  string
 	discoveryTtl   chan struct{}
-	ServerContext  string
+	ServerURLBase  string
```

Comment:
- [ ] Like in file _config.go_, this parameter should come after the "ServicePort" field above at line 20.  
  I'm sorry that I missed this detail the first time through.